config/default/manager_image_patch.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     spec:
       containers:
       # Change the value of image field below to your controller image URL
-      - image: kfserving-controller:latest
+      - image: gcr.io/kfserving/kfserving-controller-head:latest
```

Comment:
Do we need the suffix ? Tag latest means head right ?

Reply:
Yeah I'll switch to :latest-devel
:latest typically means latest stable.